packages/schematics/angular/workspace/files/package.json.template

```diff
@@ -36,8 +36,8 @@
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
     "protractor": "~7.0.0",<% } %>
-    "ts-node": "~8.3.0",
-    "tslint": "~6.1.0",
+    "ts-node": "~8.3.0",<% if (!minimal) { %>
```

Comment:
```diff
-    "protractor": "~7.0.0",<% } %>
-    "ts-node": "~8.3.0",<% if (!minimal) { %>
+    "protractor": "~7.0.0",
+    "ts-node": "~8.3.0",
     "tslint": "~6.1.0",<% } %>
```

`ts-node` is only required for `protractor`. Hence it can safely be removed when minimal is `true.`

Reply:
Done. Now `ts-node` is placed inside of `if` block.

PS: commit message body was updated.